### PR TITLE
Optimize ShortcutsDialog style

### PR DESCRIPTION
**Pull request by Dreace**
1. Fix Chinese display unnecessary line breaks.
2. Keep all row heights consistent.

Before:
image
After:
image

### CODE REVIEW

File: src/components/Island.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .Island {
   --padding: 0;
+  word-break: keep-all;
```
Comment (lipis) [inline src/components/Island.css:3]: could you add this only to `ShortcutKey` element?

https://github.com/excalidraw/excalidraw/blob/master/src/components/ShortcutsDialog.tsx#L88-L93

Comment (Dreace) [reply src/components/Island.css]: Already edited.

File: src/components/ShortcutsDialog.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const ShortcutIsland = (props: {
       width: "49%",
       border: "1px solid #ced4da",
       marginBottom: "16px",
+      height: "-webkit-fill-available",
```
Comment (lipis) [inline src/components/ShortcutsDialog.tsx:16]: I have some ideas on changing that in the future.. but it's good for now.

**Comment by lipis** [comment]
Thanks!

**Pull request merged**